lib/osut/utils.rb

```diff
@@ -256,14 +640,14 @@ def genConstruction(model = nil, specs = {})
       d  = 0.015
       a[:compo][:mat] = @@mats[mt]
       a[:compo][:d  ] = d
-      a[:compo][:id ] = "OSut|#{mt}|#{format('%03d', d*1000)[-3..-1]}"
+      a[:compo][:id ] = "OSut:#{mt}:#{format('%03d', d*1000)[-3..-1]}"
     when :partition
       unless specs[:clad] == :none
         d  = 0.015
         mt = :drywall
         a[:clad][:mat] = @@mats[mt]
         a[:clad][:d  ] = d
-        a[:clad][:id ] = "OSut|#{mt}|#{format('%03d', d*1000)[-3..-1]}"
+        a[:clad][:id ] = "OSut:#{mt}:#{format('%03d', d*1000)[-3..-1]}"
```

Comment:
Switching string separating chars, from pipe "|" to colon ":". Inserting pipe chars is not a great idea.